runtime/src/iree/vm/bytecode_module.c

```diff
@@ -100,46 +236,27 @@ static iree_status_t iree_vm_bytecode_module_resolve_types(
   return status;
 }
 
-// Computes the total length of the flatbuffer and the base offset for any
-// concatenated rodata.
-static iree_host_size_t iree_vm_bytecode_module_flatbuffer_rodata_offset(
-    iree_const_byte_span_t flatbuffer_data) {
-  if (flatbuffer_data.data_length < sizeof(flatbuffers_uoffset_t)) return 0;
-  size_t external_rodata_offset = 0;
-  flatbuffers_read_size_prefix((void*)flatbuffer_data.data,
-                               &external_rodata_offset);
-  external_rodata_offset += sizeof(flatbuffers_uoffset_t);
-  return iree_host_align(external_rodata_offset, 128);
-}
-
-// Verifies the structure of the flatbuffer so that we can avoid doing so during
+// Verifies the structure of the FlatBuffer so that we can avoid doing so during
 // runtime. There are still some conditions we must be aware of (such as omitted
 // names on functions with internal linkage), however we shouldn't need to
-// bounds check anything within the flatbuffer after this succeeds.
+// bounds check anything within the FlatBuffer after this succeeds.
```

Comment:
nit: I see other comments using lowercase `flatbuffer`. Want to adjust all of those too?

Reply:
did a cleanup across the project